rush.json

```diff
@@ -632,6 +632,11 @@
       "projectFolder": "sdk/digitaltwins/digital-twins-core",
       "versionPolicyName": "client"
     },
+    {
+      "packageName": "@azure/perf-test-storage-blob",
```

Comment:
If we're moving forward with the approach of using little micro-packages that are rush-managed, can we put the "perf-test" in the namespace part of the name? I'm just thinking out loud that it will be useful for scripting/bulk processing if these are `@azure-perf-test/storage-blob` than otherwise. It'll make it easiest to query package names for specific kinds of packages, since we will eventually have hundreds of packages as we add more services and more perf tests. Maybe we could rename `dev-tool`, `eslint-plugin-azure-sdk`, `test-utils-recorder`, and the other internal tools to use an `@azure-tools` namespace as well.

Just something to consider.

Reply:
Very good idea. I love `@azure-perf-test/storage-blob`. 
I'll make the change in this PR, if anyone has objections, please comment on this.

Reply:
A couple of things came up while making this change.
`rush` strips out the namespace name and saves it as this.
![image](https://user-images.githubusercontent.com/10452642/108412694-f14d8580-71de-11eb-8499-b2d854a347a7.png)
Not that it doesn't work, but something to consider. `storage-blob.tgz` and `storage-blob-1.tgz` can be confusing.

And I can no longer build storage-blob with `rush build -t storage-blob`, we must use `rush build -t @azure/storage-blob` to build storage-blob SDK which would break the Dev Exp.

Reply:
@azure-tests/perf-storage-blob?

Reply:
I like that one too. Feels like a hack, but I don't have any objections.
Going with `@azure-tests/perf-storage-blob`.